packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.tsx

```diff
@@ -466,9 +466,16 @@ const ChallengeTeamWizard: React.FunctionComponent<
     onClose()
   }
 
-  const onConfirmHandlerMap: Record<string, () => Promise<void>> | void = {
+  const onConfirmHandlerMap: Record<
+    string,
+    (() => Promise<void>) | (() => undefined)
+  > | void = {
     CREATE_NEW_TEAM: handleCreateTeam,
     JOIN_REQUEST_FORM: handleRequestMembership,
+    JOIN_REQUEST_SENT: () => {
```

Comment:
The JOIN_REQUEST_SENT didn't have a confirmation handler.  It should just hide the dialog.